### ISSUE TITLE
Cannot run: ErrorId 1799

### ISSUE THREAD

**Issue by dotKokott** [fbarresi/BeckhoffHttpClient#9]
Hello!

First of all thank you for writing an open source alternative HttpClient.

I have a problem running it. When I try to do a GET to a local server I get a: 1799 -> DeviceNotReady error id.

![image](https://user-images.githubusercontent.com/503388/94025525-e241e300-fdb8-11ea-94b6-4073546a0f6b.png)

Any idea what I am doing wrong?

**Comment by fbarresi** [comment]
Hi!

You error Id isn't an ADS error code, but a Win32 Code and means: ERROR_INVALID_SEPARATOR_FILE - "The specified separator file is invalid"

Did you already installed the Twincat (unofficial) function on your target system?
You can check it under the path `C:\TwinCAT\Functions\Unofficial\BeckhoffHttpClient\`.

I'm wondering what can cause this error in your setup... 🤨 
Are you able get the example on the README up and running?

Best regards, 

FB

**Comment by dotKokott** [comment]
Yes the function is installed & I can see it under this path. I do have the rest of TwinCAT installed under E:\TwinCAT (not sure if that would make a difference)

The example from the README shows the same error.

**Comment by fbarresi** [comment]
🤔 I suppose it could be the problem, but I cannot reproduce it...
I would advise you to try a couple things:
- Debug the httpClient FB ( in particular its private fields: `startProcess`, `getLocalAmsNetId`, `initialized` and `startProcess.COMNDLINE`)
- import the source code fot HttpClient in your project (in this case you may also move the twincat function folder to E: and change the hardcoded paths `startProcess.PATHSTR` and `startProcess.DIRNAME`)

**Comment by fbarresi** [comment]
Hi!
Did you get all up and running?
If yes, would your share your solution in this issue?

**Comment by dotKokott** [comment]
Hey! I had to move on for now and use the original Beckhoff httpClient.
I hope I'll get around to it towards the end of this project. Should I get this running I will share my solution!
